src/test/java/duke/DeadlineTest.java

```diff
@@ -0,0 +1,16 @@
+package duke;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import duke.task.Deadline;
+
+public class DeadlineTest {
+    @Test
+    public void deadline_toString_success() {
```

Comment:
Perhaps you can consider adding more tests to check for the exceptions thrown on invalid output?

Reply:
Okay, thanks for the suggestion!